test/integration/serving/serving_test.go

```diff
@@ -291,7 +291,7 @@ func testComponent(t *testing.T, tester componentTester, kubeconfig, brokenKubec
 					t.Fatalf("failed to GET %s from component: %v", tt.path, err)
 				}
 
-				body, err := ioutil.ReadAll(r.Body)
+				body, _ := ioutil.ReadAll(r.Body)
```

Comment:
better to handle the error and and do a `t.Fatalf(....err)`

Reply:
I was hesitant to handle this error, because it didn't affect the test results (we verified the test by StatusCode).
But as you say, we'd better deal with it.